scripts/release-app-interface.sh

```
@@ -148,6 +148,24 @@ QE has verified all queued issues
 EEOOFF
 }
 
+# Tag the prod SHA in koku-ui (does not tag stage deploys).
+# Triggers .github/workflows/tag_release.yml via workflow_dispatch.
+# HCCM and ROS can be tagged independently or together.
+tagProdReleases()
+{
+  cd $KOKU_UI_DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)